pkg/commands/git_config/get_key.go

```diff
@@ -53,7 +54,9 @@ func runGitConfigCmd(cmd *exec.Cmd) (string, error) {
 
 func getGitConfigCmd(key string) *exec.Cmd {
 	gitArgs := []string{"config", "--get", "--null", key}
-	return exec.Command("git", gitArgs...)
+	cmd := exec.Command("git", gitArgs...)
+	cmd.Env = os.Environ()
```

Comment:
I don't understand why you have to do this. From the documentation of the exec package, it sounds like it should be the default to pass the current process's env to the executed command, unless you set `cmd.Env` to something else. (For `Command`, it says: `It sets only the Path and Args in the returned structure.`, which means that `cmd.Env` stays nil; and for `Cmd.Env`, it says `If Env is nil, the new process uses the current process's environment.`)

What am I missing?

Reply:
Also, your test still succeeds even if you revert your production code change, which confirms that things are already working as expected. There must be something wrong with the way you set up your environment.

Reply:
Hmm, oddly enough I'm seeing the opposite behavior. Lemme dig around some more.

Reply:
Hi @stefanhaller, you are absolutely right: my env was probably broken 2 days ago hence thinking it doesn't already work like this. Closing!